tests/snmp/test_snmp_loopback.py

```diff
@@ -0,0 +1,61 @@
+import pytest
+import time
+import logging
+import ipaddress
+try:  # python3 
+    from shlex import quote
+except ImportError:  #  python2
+    from pipes import quote
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.topology('any'),
+    pytest.mark.device_type('vs')
+]
+
+def get_snmp_output(ip, duthost, nbr, creds):
+    ipaddr = ipaddress.ip_address(ip)
+    iptables_cmd = "iptables"
+
+    # TODO : Fix snmp query over loopback v6 and remove this check and add IPv6 ACL table/rule.
+    if isinstance(ipaddr, ipaddress.IPv6Address):
+        iptables_cmd = "ip6tables"
+        return None 
+    
+    ip_tbl_rule_add = "sudo {} -I INPUT 1 -p udp --dport 161 -d {} -j ACCEPT".format(iptables_cmd, ip) 
```

Comment:
why is this iptable rule needed ?

Reply:
By default, packets to loopback interface are dropped. So to accept snmp packets over loopback IP, we have to add this rule above the default rule.
DROP       all      anywhere             fc00:1::32 /
DROP       all  --  anywhere             10.1.0.32